docs/mv-guide.md

```diff
@@ -0,0 +1,342 @@
+<!--
```

Comment:
can you use rename without adding another doc, so that it is easier to find out what is changed in this doc

Reply:
I move the file to doument root directory from index directory.
before commit, execute "mvn status" at local , display rename & change
but after commit, it changed to delete old file and add new file.